### PR TITLE
Add link to internal handbook in homepage intro

### PR DESCRIPTION
**Pull request by aduth**
## 🛠 Summary of changes

Updates the homepage to include a visually-distinct link to the internal handbook.

Currently, it's hard to know that an internal handbook even exists with additional handbook material. The intent here is to improve awareness of its existence, without being too overly distracting.

Related Slack discussion: https://gsa-tts.slack.com/archives/CEUQ9FXNJ/p1733941222366069

## 📜 Testing Plan

1. Go to preview URL
2. See alert with link to internal handbook
3. Verify that the link is functional

## 👀 Screenshots

Before|After
---|---
image|image

### CODE REVIEW

File: _plugins/view_component.rb [hunk 1]
```diff
@@ -71,7 +74,13 @@ def parse_params(context)
   end
 
   def render(context)
-    content = super.html_safe
+    content = super
+    if !content.include?('<')
+      content = context.registers[:site].
+        find_converter_instance(Jekyll::Converters::Markdown).
+        convert(super).sub(/^<p>(.+)<\/p>$/, '\1')
```
Comment (aduth) [inline _plugins/view_component.rb:81]: Using `markdownify` built-in filter as reference: https://github.com/jekyll/jekyll/blob/0e4182a/lib/jekyll/filters.rb#L11-L20

**Review by zachmargolis — APPROVED** [review]

**Comment by aduth** [comment]
Apparently I missed that we already have a link, pretty close to the top! But I think the fact that I missed it could be pretty telling in itself?

In any case, maybe it'd be a good idea to remove that extra link if we have this one now instead.

Screenshot 2024-12-11 at 3 22 23 PM

File: index.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ to [avoid contributing sensitive information][sensitive-information].
 
 [sensitive-information]: {{ site.baseurl }}/articles/contributing.html#sensitive-information
 
+{% component alert type=:info %}
+Internal documentation can be found in our [🔒 Internal Login.gov Handbook](https://lg-public.pages.production.gitlab.login.gov/identity-internal-handbook/).
```
[the remark this answers is not in the document]
Comment (aduth) [reply index.md:16]: >how come this lock emoji doesn't get duplicated from private-eye?

That's a good question 🤔

**Comment by zachmargolis** [comment]
> Apparently I missed that we already have a link, pretty close to the top! But I think the fact that I missed it could be pretty telling in itself?
> 
> In any case, maybe it'd be a good idea to remove that extra link if we have this one now instead.

It's "below the fold" on my screen so I think it's fine to leave the duplication

File: index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zachmargolis) [reply index.md:16]: oh wait it is on the preview branch, but not in your screenshot

Screenshot 2024-12-11 at 12 27 30 PM

```suggestion
Internal documentation can be found in our [Internal Login.gov Handbook](https://lg-public.pages.production.gitlab.login.gov/identity-internal-handbook/).
```

**Comment by gina-yamada** [comment]
I noticed lock is slightly different- see screenshot.  Not blocking.
Link takes me to internal hb when signed in, incognito window takes me to login in. 
LGTM

Screenshot 2024-12-11 at 2 24 46 PM

**Review by gina-yamada — APPROVED** [review]

**Pull request merged**